### PR TITLE
Third task

### PR DESCRIPTION
**Pull request by Kalvarado07**
# Pull Request
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes  ([Trello](https://trello.com/c/70cQeMSD))
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## How Has This Been Tested?
Tested on chroem and edge

### CODE REVIEW

File: OnePiece/assets/css/style.css [hunk 1]
```diff
@@ -1,52 +1,859 @@
...
+}
+.nav__menu__lang {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  align-items: left;
+  padding: 0px;
+  gap: 8px;
+  width: 169px;
+  height: 30.79px;
```
Comment (kduranaraya) [inline OnePiece/assets/css/style.css:70]: Is not necessary use .00 values. Use 31px o 30px. The different between some of these options and 30.79px is minimal, but the code can be clearer.

File: OnePiece/assets/css/style.css [hunk 2]
```diff
@@ -1,52 +1,859 @@
...
+  text-decoration: none;
+}
+
+#nav__checkbox:checked ~ ul.nav__menu li {
+  display: block;
+}
+
+#nav__checkbox:checked ~ ul.nav__menu {
+  box-shadow: 0px 4px 3px #F3F3F3;
+  padding-bottom: 34.21px;
```
Comment (kduranaraya) [inline OnePiece/assets/css/style.css:93]: Same here. Use 34px

File: OnePiece/assets/css/style.css [hunk 3]
```diff
@@ -1,52 +1,859 @@
...
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  gap: 16px;
+}
+.wrapper__content-section {
+  width: 95%;
+}
+.wrapper__char-card {
+  width: 181.94px;
```
Comment (kduranaraya) [inline OnePiece/assets/css/style.css:267]: Same here.

File: OnePiece/assets/css/style.css [hunk 4]
```diff
@@ -1,52 +1,859 @@
...
+  height: 231px;
   left: 0px;
   top: 0px;
   mix-blend-mode: multiply;
   border-radius: 20.5295px;
```
Comment (kduranaraya) [inline OnePiece/assets/css/style.css:272]: Same here. Please check and change all values like this.

File: OnePiece/assets/css/style.css (hunk 1 above)
Comment (Kalvarado07) [reply OnePiece/assets/css/style.css]: Thank you for the feedback, the change was implemented.

File: OnePiece/assets/css/style.css (hunk 3 above)
Comment (Kalvarado07) [reply OnePiece/assets/css/style.css]: Change made

**Review by kduranaraya — APPROVED** [review]

**Pull request merged**